### ISSUE TITLE
Chat message timestamp has insufficient contrast

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#725]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 1.4.3 (AA) Contrast (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum).

Good contrast between text and background is important for readability, especially for people with impaired vision, as well as while browsing in bright environments.

Chat message timestamps have insufficient contrast levels:
![Message timestamp.](https://github.com/user-attachments/assets/3430110b-860a-44f8-9d1e-5a79e8181680)

The same is true for quoted message text.

**To Reproduce**
Look at the time stamp of a chat message.

**Expected behavior**
The color contrast between text and background should be at least 4.5:1, or 3:1 if the text is large (meaning font size 24px and up, or 19px and bold).

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
